aim/web/ui/src/pages/Scatters/Scatters.tsx

```diff
@@ -0,0 +1,190 @@
+import React from 'react';
+import { isEmpty } from 'lodash-es';
+
+import Table from 'components/Table/Table';
+import ChartPanel from 'components/ChartPanel/ChartPanel';
+import NotificationContainer from 'components/NotificationContainer/NotificationContainer';
+import BusyLoaderWrapper from 'components/BusyLoaderWrapper/BusyLoaderWrapper';
+import EmptyComponent from 'components/EmptyComponent/EmptyComponent';
+import TableLoader from 'components/TableLoader/TableLoader';
+import ChartLoader from 'components/ChartLoader/ChartLoader';
+import ResizePanel from 'components/ResizePanel/ResizePanel';
+
+import { ResizeModeEnum } from 'config/enums/tableEnums';
+import { RowHeightSize } from 'config/table/tableConfigs';
+import GroupingPopovers from 'config/grouping/GroupingPopovers';
+
+import Grouping from 'pages/components/Grouping/Grouping';
+import AppBar from 'pages/Metrics/components/MetricsBar/MetricsBar';
+import Controls from 'pages/Scatters/components/Controls/Controls';
+import SelectForm from 'pages/Scatters/components/SelectForm/SelectForm';
+
+import { IScattersProps } from 'types/pages/scatters/Scatters';
+
+import { ChartTypeEnum } from 'utils/d3';
+
+import './Scatters.scss';
+
+function Scatters(
+  props: IScattersProps,
+): React.FunctionComponentElement<React.ReactNode> {
+  const chartProps: any[] = React.useMemo(() => {
+    return (props.scatterPlotData || []).map((data: any, index: number) => ({
+      chartTitle: props.chartTitleData[index],
+    }));
+  }, [props.scatterPlotData, props.chartTitleData]);
+
+  return (
+    <div ref={props.wrapperElemRef} className='Scatters__container'>
+      <section className='Scatters__section'>
```

Comment:
For every page we have exactly one container and we have them `Metrics_container` and `Scatters_container`  etc. because styles overrides when we go from one page to another